src/Facets/AllBridgeFacet.sol

```diff
@@ -128,4 +193,41 @@ contract AllBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
 
         emit LiFiTransferStarted(_bridgeData);
     }
+
+    /// @notice Converts LiFi internal chain IDs to AllBridge chain IDs
+    /// @param _destinationChainId The LiFi chain ID to convert
+    /// @return The corresponding Chainflip chain ID
+    /// @dev Reverts if the destination chain is not supported
+    function _getAllBridgeChainId(
+        uint256 _destinationChainId
+    ) internal pure returns (uint256) {
+        // split possible values in half for more efficient search/matching
```

Comment:
Can you please explain this comment.

Reply:
this comment was outdated from a different, previous approach. Removed.